common/lumens.js

```diff
@@ -50,13 +50,20 @@ const accounts = {
 export const ORIGINAL_SUPPLY_AMOUNT = "100000000000";
 
 export function getLumenBalance(horizonURL, accountId) {
-  return axios.get(`${horizonURL}/accounts/${accountId}`).then((response) => {
-    var xlmBalance = find(
-      response.data.balances,
-      (b) => b.asset_type == "native",
-    );
-    return xlmBalance.balance;
-  });
+  return axios
+    .get(`${horizonURL}/accounts/${accountId}`)
+    .then((response) => {
+      var xlmBalance = find(
+        response.data.balances,
+        (b) => b.asset_type == "native",
+      );
+      return xlmBalance.balance;
+    })
+    .catch((error) => {
+      if (error.response && error.response.status == 404) {
+        return "0.0"; // disconsider the balance of an account if there's an error with the request
```

Comment:
💡 Suggested rewording of this comment. I don't think disconsider is a word, and I think we can be specific about the cases this occurs in.
```suggestion
        return "0.0"; // consider the balance of an account zero if the account does not exist or has been deleted from the network
```

Reply:
💯 good catch :) I meant disregard, but I like your phrasing better.